### PR TITLE
Init (Ai): add basic client

### PR DESCRIPTION
**Pull request by RoheeAxel**
This PR does the following:

• added basic python client to connect to the server
• added class Agent who is basic agent with funtion to call server
• added class Repetoile who is basic agent who can survive

The code has been tested and no problems were found on my side.

### CODE REVIEW

**Review by Sinan-Karakaya — APPROVED** [review]
Seems good

**Review by JasonLAGOUTE — APPROVED** [review]
Good code :)

File: zappy_ai/src/Agent/Agent.py [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+            return (0, None)
+
+        direction = None
+        xToGo = caseCenter - listIndex
+        if xToGo < 0:
+            xToGo = -xToGo
+            direction = "Right"
+            return (xToGo, direction)
+
+        xToGo = xToGo
```
Comment (thomasjuin1) [inline zappy_ai/src/Agent/Agent.py:148]: Can you explain this ?

**Review by thomasjuin1 — COMMENTED** [review]
Seems fine, the doc is good

**Review by thomasjuin1 — APPROVED** [review]

**Pull request merged**